src/Schema/Grammar.php

```
@@ -50,6 +50,27 @@ public function compileTables($schema)
         return 'select `table_name` as name, `table_type` as type, `parent_table_name` as parent from information_schema.tables where table_schema = \'\' and table_type = \'BASE TABLE\'';
     }
 
+    /**
+     * Compile the query to determine the columns.
+     *
+     * @param string $table
+     * @return string
+     */
+    public function compileColumns($table)
```

PHPStan (2.0.3)

Method Colopl\Spanner\Schema\Grammar::compileColumns() overrides method Illuminate\Database\Schema\Grammars\Grammar::compileColumns() but misses parameter #2 $table.


(parameter.missing)